tests/integration_tests/document/test_update.py

```diff
@@ -58,8 +54,7 @@ def test_update_document_super(
     data = response.json()
     assert data["import_id"] == "D.0.0.2"
     assert data["variant_name"] == "Translation"
-    assert data["metadata"] == {"role": ["SUMMARY"]}
-    assert data["type"] == "Annex"
+    assert data["metadata"] == {"role": ["SUMMARY"], "type": ["Annex"]}
```

Comment:
Mayne we can pullout a `DEFAULT_DOC_META` constant? Which would make subsequent changes a little easier?

Reply:
Will do this in the next PR :)